packages/wrangler/src/__tests__/pages/publish.test.ts

```diff
@@ -0,0 +1,2168 @@
+/* eslint-disable no-shadow */
```

Comment:
Is there any way it would make sense to break these tests down into separate files? (No worries if not)

Reply:
these are all `publish` related tests. I don't have a good idea yet on how we can better organise these. did you have smth in mind?

Reply:
No I didn't... This is great as-is :)